package.json

```
@@ -19,10 +19,12 @@
     "ableton:copy-script": "set -- ~/Music/Ableton/User\\ Library/Remote\\ Scripts && mkdir -p \"$1\" && rm -rf \"$1/AbletonJS\" && cp -r \"$(pwd)/midi-script\" \"$1/AbletonJS\" && rm -rf \"$1/AbletonJS/_Framework\"",
     "ableton10:launch": "set -- /Applications/Ableton*10* && open \"$1\"",
     "ableton11:launch": "set -- /Applications/Ableton*11* && open \"$1\"",
+    "ableton12:launch": "set -- /Applications/Ableton*11* && open \"$1\"",
```

This should be `Ableton*12*` I think:

```suggestion
    "ableton12:launch": "set -- /Applications/Ableton*12* && open \"$1\"",
```